README.md

```diff
@@ -18,10 +18,12 @@ resource "aws_instance" "default" {
 ```
 module "tf_ami_from_instance" {
   source             = "git::https://github.com/cloudposse/tf_ami_from_instance.git?ref=master"
-  name               = "${aws_instance.web.id}"
   source_instance_id = "${aws_instance.web.id}"
```

Comment:
`source_` is redundant. Rename `source_instance_id` to `instance_id`

Reply:
`source_instance_id` is inherited from resource `aws_ami_from_instance`. 
Let's use `source_instance_id`

Reply:
Ya, makes sense. @s2504s  is just passing it through... https://github.com/cloudposse/tf_ami_from_instance/pull/4/files#diff-7a370d8342e7203b805911c92454f0f4R13